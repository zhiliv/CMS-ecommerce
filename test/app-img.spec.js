import { shallowMount } from '@vue/test-utils'
import P from '@/components/app/img/img.vue'

describe('app-img.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(P, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        src: 'img.jpg'
      },
      slots:{
        default: 'Содержимое'
      }
    })
  })
  it('Содержимое html для app-img ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
