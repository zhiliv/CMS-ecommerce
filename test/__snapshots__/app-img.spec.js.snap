// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-img.js Содержимое html для app-img  1`] = `
<img
  class="simple simple-xl"
  id="indent"
  src="img.jpg"
  style="background: white;"
/>
`;
